### PR TITLE
ghostscript: update to v9.54.0

### PR DESCRIPTION
**Pull request by nickgaya**
#### Description

* Update version to 9.54.0
    * Disable tesseract/leptonica (see https://trac.macports.org/ticket/62832)
* Clean up outdated patches

###### Type(s)
- [x] update

###### Tested on
macOS 10.14.6 18G9028 x86_64
Xcode 11.3.1 11C505

###### Verification
Have you

- [x] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [x] squashed and [minimized your commits](https://guide.macports.org/#project.github)?
- [x] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [x] checked your Portfile with `port lint`?
- [x] tried a full install with `sudo port -vst install`?
- [x] tested basic functionality of all binary files?

### CODE REVIEW

File: print/ghostscript/Portfile [hunk 1]
```diff
@@ -40,13 +40,10 @@ patchfiles          patch-base_unix-dll.mak.diff \
 # https://trac.macports.org/ticket/61306
 patchfiles-append    patch-base_fapi_ft.c.diff
 
-# Missing stdlib.h for abs()
-patchfiles-append    patch-jpegxr_r_strip.c.diff
```
Comment (nickgaya) [inline print/ghostscript/Portfile:44]: This patch was included in 9.54.0 so deleting it here.

**Review by chrstphrchvz — CHANGES_REQUESTED** [review]
I believe the patch added in 90e528935b can be dropped as well: the upstream report  https://bugs.ghostscript.com/show_bug.cgi?id=702985 was resolved by ArtifexSoftware/ghostpdl@41ef9a0bc3

**Comment by chrstphrchvz** [comment]
CI build failure for `ghostscript-fonts-hiragino` is expected: https://trac.macports.org/ticket/54098

**Comment by nickgaya** [comment]
@chrstphrchvz You're right, good catch.  After scrutinizing the build more closely I found several other things I had missed initially. See the updated commit message for a full list of changes and rationale.

**Comment by l2dy** [comment]
Merged, thank you!

**Pull request merged**